handler/core/src/main/java/io/knotx/fragments/handler/action/CircuitBreakerActionFactory.java

```diff
@@ -45,28 +49,33 @@ public String getName() {
   }
 
   @Override
-  public Action create(String alias, JsonObject config, Vertx vertx, Action doAction) {
-    if (doAction == null) {
+  public Action create(ActionConfig config, Vertx vertx) {
```

Comment:
You can introduce the `LoggableAction` interface with the default method creating the logger instance.

Reply:
I cannot see any benefits.. I left the factory method